tests/unit/aiplatform/test_metadata_schema.py

```diff
@@ -764,12 +796,22 @@ def test_classification_metrics_constructor_parameters_are_set_correctly(self):
             metadata=_TEST_UPDATED_METADATA,
         )
         expected_metadata = {
-            "test-param1": 2.0,
-            "test-param2": "test-value-1",
-            "test-param3": False,
-            "auPrc": 1.0,
-            "auRoc": 2.0,
-            "logLoss": 0.5,
+            "test-param1": _TEST_UPDATED_METADATA["test-param1"],
+            "test-param2": _TEST_UPDATED_METADATA["test-param2"],
+            "test-param3": _TEST_UPDATED_METADATA["test-param3"],
+            "aggregationType": aggregation_type,
+            "aggregationThreshold": aggregation_threshold,
+            "recall": recall,
+            "precision": precision,
+            "f1Score": f1_score,
+            "accuracy": accuracy,
+            "auPrc": au_prc,
+            "auRoc": au_roc,
+            "logLoss": log_loss,
+            "confusionMatrix": confusion_matrix.to_dict(),
+            "confidenceMetrics": [
+                confidence_metric.to_dict() for confidence_metric in confidence_metrics
+            ],
```

Comment:
could you add a negative test case to validate the Errors in Schema and Util file. see my comment above.